docs/stdlib/json.rst

```diff
@@ -132,26 +132,24 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
         db> select <bool>to_json('true');
         {true}
 
-    Note that a :eql:type:`json` value can be cast into a :eql:type:`str`
-    only when it is a JSON string.  Therefore, while the following will work
-    as expected:
+    A :eql:type:`json` type can also be casted as a :eql:type:`str` type,
+    but only when recognised as a JSON string:
 
     .. code-block:: edgeql-repl
 
         db> select <str>to_json('"something"');
         {'something'}
 
-    The operation below (casting a JSON array of
-    string ``["a", "b", "c"]`` to a *str*) will result in an error:
+    Casting a string array of JSON (``["a", "b", "c"]``) to a :eql:type:`str`
```

Comment:
```suggestion
    Casting a JSON array of strings (``["a", "b", "c"]``) to a :eql:type:`str`
```
Even though we do start with a string containing a value that can be cast to JSON as an array, the cast this is referring to is the `json` to `str` cast. That is a cast of a JSON array of strings rather than a string array of JSON.